### DISCUSSION TITLE
How to get intent by setting AndroidManifest.xml ?

### DISCUSSION THREAD

**Discussion by hoaohoao** [beeware/toga#2791] [Q&A] [answered]
I want to receive intent from other app, like FireFox or Youtube.
I modified  AndroidManifest.xml  and add 
```
<intent-filter>
      <action android:name="android.intent.action.VIEW" />
      <category android:name="android.intent.category.DEFAULT" />
      <category android:name="android.intent.category.BROWSABLE" />
      <data
          android:host="www.youtube.com"
          android:scheme="https" />
      <data
          android:host="m.youtube.com"
          android:scheme="https" />
</intent-filter>
```
, then build and run ,  my app isn't on FireFox sharing List. 
Something I missing?

**Comment by freakboy3742** [comment, marked as answer]
I can't offer any special insight here. This is a Toga support forum, not a general Android support forum.

I suggest taking this question up with an Android support forum. If, as a result of whatever debugging advice you receive, you discover that Toga (or, more likely, Briefcase) is doing something incorrectly, provide details and we can investigate further - but until we have a specific report of a problem *that is caused by Toga*, there is a limit to what we can do to help.

**Reply by mhsmith** [reply]
Answered at https://stackoverflow.com/questions/78914460.
